devtools/onion.c

```diff
@@ -12,32 +13,55 @@
 #include <unistd.h>
 
 #define ASSOC_DATA_SIZE 32
+#define PUBKEY_LEN 33
+#define PRIVKEY_LEN 32
```

Comment:
`PUBKEY_CMPR_LEN` from bitcoin/pubkey.h?  Hmm we don't have a convenient PRIVKEY_LEN though...

If you're bored you could add one to bitcoin/privkey.h :)